File: atcoder/abc143C_slimes.ml
(* Vicfred
 * https://atcoder.jp/contests/abc143/tasks/abc143_c
 * implementation
 * *)
let main =
    Scanf.scanf "%d\n%s" @@ fun n s ->
        Printf.printf "%d\n" @@
        Array.fold_left ( + ) 1 @@
        Array.init (n - 1) @@ fun i ->
            if s.[i] = s.[i + 1] then 0 else 1

File: atcoder/abc143C_slimes.cpp
// uninhm
// https://atcoder.jp/contests/abc143/tasks/abc143_c
// implementation

#include<iostream>
#include<stdio.h>

using namespace std;

int main() {
  int n;
  scanf("%d\n", &n);

  char c, last;
  scanf("%c", &c);
  last = c;

  int ans = 1;
  for(int i = 1; i < n; ++i) {
    cin >> c;
  
    if(c != last)
      ans++;

    last = c;
  }

  cout << ans << endl;
}
